js_modules/dagit/packages/core/src/asset-graph/useLiveDataForAssetKeys.tsx

```diff
@@ -187,6 +187,9 @@ const ASSETS_GRAPH_LIVE_QUERY = gql`
   query AssetGraphLiveQuery($assetKeys: [AssetKeyInput!]!) {
     assetNodes(assetKeys: $assetKeys, loadMaterializations: true) {
       id
+      type {
+        name
+      }
```

Comment:
I'm not totally sure this ended up being used in this PR?

Reply:
This is how we determine that it's dynamic

Reply:
oh wait, you're right, not using this one.